### ISSUE TITLE
Attach binary downloads to releases on GitHub?

### ISSUE THREAD

**Issue by svenevs** [doxygen/doxygen#6705]
Hello, I develop a tool that works with doxygen.  For CI I used to download the source from what appears to be the mirror site (e.g., `https://www.stack.nl/~dimitri/doxygen/manual/install.html`).  Recently this has been giving 403 so I started just getting the archives from GitHub and building from source on Linux and Windows (on windows I started having trouble with the two available from `choco`), and on OSX I still use `brew` to install the bottled version.

I'm in the process of updating my documentation links to `doxygen.nl` (which is where `doxygen.org` redirects to).  At the same time I realized that there are binary installers that I can use for both Linux and Windows available ([from here](http://www.doxygen.nl/download.html)).  However, the download speeds are slower than just getting the source distribution from GitHub and compiling from source.

With tagged releases, it is also possible to attach additional downloads to the release, in particular `doxygen-X.Y.Z.linux.bin.tar.gz`, the windows installers, and the dmg if desired (though I think `brew` should still be preferred).  I don't know how complicated this would make packaging for the maintainers (this would mean potentially 5 additional uploads: linux, mac, windows x 3), but it has a couple of nice benefits:

1. The `doxygen.nl` site won't need to handle as many requests if people start downloading the binary distributions from GitHub.
2. Users downloading from the GitHub binary releases will benefit from GitHub's pervasive CDN and will likely experience much faster download speeds.

As a reference point, KitWare recently moved their CMake downloads for these reasons.  The old download URLs are still available (I think it was like `cmake.org/Files` or something), but [their downloads page](https://cmake.org/download/) actually redirects to the GitHub downloads now.  This was done after one of the developers solicited feedback on some bandwidth tests, and the GitHub speeds were _at least_ twice as fast for everybody, regardless of geographic location (all timezones in the United States as well as various data points across Europe were tested).

If this sounds like something that you would like to do, I would be more than happy to download / re-upload the binary distributions to GitHub for however many previous versions of doxygen you would like.  Of course I've never contributed here / you don't know me ... so I understand if you would rather not do this or delegate to somebody else.  If you would like me to do it, I would need access to the repo ... and ideally some md5 or sha sums to verify everything.

Basically, please let me know if there is anything I can do to help make this available assuming you want it!

**Comment by svenevs** [comment]
Well...if you want to do it I am happy to help still.  But I overlooked the following sentence at the end:

> ### Other releases
> 
> All releases of doxygen can be found at http://sourceforge.net/projects/doxygen/files/.

Sorry for the noise, these downloads are much faster than `doxygen.nl` and I will use them.
